docs/samples/index.md

```diff
@@ -1,7 +1,10 @@
-# Usage samples
+# Samples
 
-## Table of contents
+## Usage Samples
+
+1. [Running in a Python pipeline](python/index.md)
```

Comment:
This links are currently not working... need to verify they are.

Reply:
I've checked again and the links worked. where did u validate this?

Reply:
in github readme... might be that's the problem.